### PR TITLE
XW-199 | Add Polldaddy widget

### PR DESCRIPTION
**Pull request by vforge**
https://wikia-inc.atlassian.net/browse/XW-199
https://github.com/Wikia/app/pull/8513/files

### CODE REVIEW

File: front/scripts/main/components/WidgetPolldaddyComponent.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+	classNames: ['widget-polldaddy'],
+	layoutName: 'components/widget-polldaddy',
+	data: null,
+
+	didInsertElement(): void {
+		this.loadScript();
+	},
+
+	loadScript(): void {
+		var id = this.get('data').id;
```
Comment (rogatty) [inline front/scripts/main/components/WidgetPolldaddyComponent.ts]: `this.get('data.id')`

**Comment by rogatty** [comment]
LGTM
Should we also remove `PollDaddyMixin`?

**Comment by Warkot** [comment]
LGTM

**Comment by vforge** [comment]
@rogatty  https://wikia-inc.atlassian.net/browse/XW-366

**Comment by bognix** [comment]
LGTM

**Pull request merged**